App.js

```diff
@@ -34,22 +36,43 @@ export default function App() {
   if (!global.atob) {
     global.atob = decode;
   }
+  const ref = React.useRef();
+
+  
+  // connecting password reset url to PasswordResetScreen
+  const { getInitialState } = useLinking(ref, {
+    prefixes: [prefix],
+    config: {
+      ResetPassword: "resetPassword/reset/:token",
+    },
+  });
+
+  const [isReady, setIsReady] = React.useState(false);
+  const [initialState, setInitialState] = React.useState();
+
+  React.useEffect(() => {
+    getInitialState()
+      .catch(() => {})
+      .then(state => {
+        if (state !== undefined) {
+          setInitialState(state);
+        }
+
+        setIsReady(true);
+      });
+  }, [getInitialState]);
+
+  if (!isReady) {
+    return null;
+  }
+
   return (
     <AuthContext.Provider value={{ user, setUser }}>
       <PaperProvider>
-        <NavigationContainer>
-          {user ? <AccountScreen /> : <AuthNavigator test={false} />}
```

Comment:
Did you purposefully remove the "test" prop from AuthNavigator or is that a merge/rebase side effect? Same question for styles below.